bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/runtime/CodeCheckEnvironmentChecker.java

```diff
@@ -1,34 +1,42 @@
 package org.codechecker.eclipse.plugin.runtime;
```

Comment:
A large part of modification is related to this class. Paths in Shell command scripts got misinterpreted before. This was corrected with the usage of String - File Command substitution maps.